src/script/conversation/ConversationRepository.ts

```diff
@@ -2255,7 +2255,7 @@ export class ConversationRepository {
    */
   private async clearConversationContent(conversation: Conversation, timestamp: number) {
     await this.deleteMessages(conversation, timestamp);
-    await this.addCreationMessage(conversation, !!this.userState.self()?.isTemporaryGuest(), timestamp);
+    await this.addCreationMessage(conversation, !!this.userState.self()?.isTemporaryGuest());
```

Comment:
This is the actual fix, 0 is a default timestamp param for `.addCreationMessage` method.